src/engine/enginedeck.cpp

```diff
@@ -56,6 +56,7 @@ EngineDeck::EngineDeck(QString group,
 
     // Set up additional engines
     m_pPregain = new EnginePregain(group);
+    m_pVinylSoundEmu = NULL;
```

Comment:
since we got rid of vinysoundemu completely, might as well blow away all references to this.